surge/projects.py

```diff
@@ -61,7 +61,8 @@ def create(cls,
                num_workers_per_task: int = 1,
                tags=[],
                carousel=None,
-               template_id=None):
+               template_id=None,
+               description=""):
```

Comment:
I think we should default to None here

also add type annotation `:str` 

I think that should be the type annotation for `template_id` as well, not as sure on some of the other un annotated args